sscolari: Essay proposal, Task 1 (11 April, 17:00)

stefanoscolapasta opened the pull request:
# Assignment Proposal

## Title

Cloud Gaming and Containers. A comparison and evaluation of performance between Containers and Virtual Machines.

## Names and KTH ID

  - Stefano Scolari (sscolari@kth.se)

## Deadline

- Task 1

## Category

- Essay

## Description

In the essay I will give a short introduction of Cloud Gaming. <br/>Following, I will present differences in performance and usage for Cloud Gaming between Containers and Virtual Machines as the cloud servers use virtualization techniques to isolate users and share resources among dedicated servers.<br/>
Although leading Cloud Gaming to great benefit, containers were shown to improve also Game Development itself, for example by making the game server's architecture simpler and more scalable.<br/>
Whilst great emphasis is put into comparing the performance differences between the two types of virtualizationon, and also explaining how these can affect the Quality of Service and Quality of Experience, reflections will be made on their usefulness and integration in the currrent DevOps approach to Game Development.  

**Relevance**

Evaluating Container's performance is of great relevance and importance for DevOps, as gaining greater understanding and relevant benchmarking can help inform less innovation-prone and habitual sectors like Game Development to more extensively use and implement this kind of technology.

Commit:
Added readme

stefanoscolapasta committed:
Update README.md

stefanoscolapasta committed:
Update README.md

stefanoscolapasta committed:
Merge branch 'KTH:2023' into 2023

Commit:
Added demo README

stefanoscolapasta committed:
Update README.md

stefanoscolapasta committed:
Merge branch 'KTH:2023' into 2023

Commit:
Essay readme

stefanoscolapasta committed:
Update README.md

stefanoscolapasta committed:
Update README.md

stefanoscolapasta committed:
Update README.md

stefanoscolapasta committed:
Update README.md

Deee92 commented:
Hi @stefanoscolapasta, please remove `contributions/demo/week4/sscolari-morello/README.md` from this PR. Your PR must only propose a single task.

Deee92 commented on contributions/essay/sscolari/README.md:
```diff
@@ -0,0 +1,27 @@
+
+## Deadline
+
+- Task 1 (11 April, 17:00)
```
@stefanoscolapasta, please change line 13 to `- Task 1`

Commit:
Removed useless contr to week4

stefanoscolapasta committed:
Removed (11 april, 17:00) as requested

stefanoscolapasta commented:
> Hi @stefanoscolapasta, please remove `contributions/demo/week4/sscolari-morello/README.md` from this PR. Your PR must only propose a single task.

I have removed the folder (sorry about it, I forgot to update my forked repo) and also removed the "(11 April, 17:00)".
